src/js/components/Anchor/doc.js

```diff
@@ -2,6 +2,52 @@ import { describe, PropTypes } from 'react-desc';
 
 import { genericProps, getAvailableAtBadge } from '../../utils';
 
+export const themeDoc = {
+  'global.focus.border.color': {
```

Comment:
I'm concerned that documenting global properties within each component will make it hard to know how to adjust global properties. It seems this would require hunting through all component's documentation trying to ascertain what uses the global theme properties. I'm wondering if we should just refer to the global properties referenced but define them and their default values in a single place.

Reply:
I think they are complementary. I still think there should be a place where we document Theme and their global properties.

A lot of our components have fallbacks to a global property (e.g. control). I think it is important to describe that in each component.

The way I envision this is as follows:

1) A dedicated Theme page that will describe the structure of our theme object and an overall description of each global property.

2) Inside each component, we would describe the specific theme properties and how that component consumes the global ones. This would help users understand the fallback scenarios and common things to change in the global entry that affects that given component.

Thoughts?

Reply:
Let's try it. It will be interesting to see some details on the global stuff in action, to get a feel for it.